corporate/views.py

```diff
@@ -71,12 +71,9 @@ def check_upgrade_parameters(
     has_stripe_token: bool,
     seat_count: int,
 ) -> None:
-    if billing_modality not in VALID_BILLING_MODALITY_VALUES:
-        raise BillingError("unknown billing_modality")
-    if schedule not in VALID_BILLING_SCHEDULE_VALUES:
-        raise BillingError("unknown schedule")
-    if license_management not in VALID_LICENSE_MANAGEMENT_VALUES:
-        raise BillingError("unknown license_management")
+    assert billing_modality in VALID_BILLING_MODALITY_VALUES
+    assert schedule in VALID_BILLING_SCHEDULE_VALUES
+    assert license_management in VALID_LICENSE_MANAGEMENT_VALUES
```

Comment:
I reverted this hunk, since I think BillingError seems more appropriate than `assert`, which we aim to avoid ever using in production.